pom.xml

```diff
@@ -38,6 +38,7 @@
         <version.jetty-reactive-httpclient>1.1.5</version.jetty-reactive-httpclient>
         <version.jersey1>1.19.4</version.jersey1>
         <version.jersey2>3.0.0</version.jersey2>
+        <version.cxf>3.4.2</version.cxf>
```

Comment:
I think `<version.apache.cxf>3.4.2</version.cxf>` would be better. It would be more consistent with the other apache http client properties such as `<version.apache.httpclient>4.5.13</version.apache.httpclient>`, what do you think?